src/main/java/com/cloudbees/jenkins/support/impl/ThreadDumps.java

```diff
@@ -273,12 +273,16 @@ public static void printThreadInfo(PrintWriter writer, ThreadInfo t, ThreadMXBea
             x.printStackTrace(writer);
             cpuPercentage = 0;
         }
-        writer.printf("\"%s\" id=%d (0x%x) state=%s cpu=%d%%",
+
+        long time = t.getBlockedTime() + t.getWaitedTime();
```

Comment:
this value is pretty useless.   Object.wait() will show up here - and that is a normal thing.  Now being blocked is abnormal and is what we really want to know - ie how much time was spent when this thread should have run but could not due to contention.